src/cmd/policy.go

```diff
@@ -102,8 +102,13 @@ func RegoFuncGetGithubRepo(ctx rego.BuiltinContext, a, b *ast.Term) (*ast.Term,
 		return nil, err
 	}
 
-	if org == "" || repo == "" {
-		log.Error().Msgf("Please provide a valid org and repo")
+	if org == "" {
+		log.Error().Msgf("Please provide a valid org")
```

Comment:
is the customer going to know this is for "github" ?  I think just logging this they probably won't as it won't attribute to the policy line.

Maybe we should return this an error?

Reply:
Very true, didn't think about that.  Should I add some generic prefix to the error messages?  e.g. `opslevel.repo.github(<org>, <repo>) failed: Please provide a valid org?` Not sure how explicit we want to get?

Reply:
that or see if rego does this by changing it to be

```
if org == "" {
	return nil, fmt.Errorf("Please provide a valid value for github org")
}
```

Reply:
I tried that earlier and it doesn't output it for some reason.  Probably something I would need to change in the eval?  I can take a look though.

Reply:
ok well if there is nothing built in - we just need to make sure we give as much context as possible to help debug the issue.  Terse error messages are bad and they are even worse in a black box system like we are building with "repo analysis"

Reply:
For sure, I'll add the function call that way it can be at least be identified in the policy.